redash/models/__init__.py

```diff
@@ -644,6 +645,43 @@ def outdated_queries(cls):
 
         return list(outdated_queries.values())
 
+    @classmethod
+    def _do_multi_byte_search(cls, all_queries, term, limit=None):
+        # term examples:
+        #    - word
+        #    - name:word
+        #    - query:word
+        #    - "multiple words"
+        #    - name:"multiple words"
+        #    - word1 word2 word3
+        #    - word1 "multiple word" query:"select foo"
+        tokens = re.findall(r'(?:([^:\s]+):)?(?:"([^"]+)"|(\S+))', term)
```

Comment:
Is there a better way to break _words_ and _"multiple words"_ into tokens? This regex appears to work, may also contain hard-to-find edge cases

Reply:
@eradman 

I see. Thank you !